authors/apps/authentication/tests/test_password_reset.py

```diff
@@ -71,10 +71,25 @@ def test_null_email(self):
         self.assertEqual(
             self.contains_error(
                 response.data.get("errors").get("email"),
-                "This field may not be null."
+                "Please provide your email address"
             ), True
         )
 
+    def test_successful_reset_link_sent(self):
+        """
+        Tests for successful sending of the password
+        reset link
+        """
+        response = self.password_reset()
+        self.assertEqual(
+            response.status_code,
+            status.HTTP_200_OK
+        )
+        self.assertEqual(
+            response.data.get("data")[0].get("message"),
+            "An email has been sent to your email address to reset your password"
```

Comment:
This message could also be better. For instance:
```
A password reset message was sent to your email address. Please click the link in that message to reset your password. 
```

Reply:
I've update it